thrift_api/report_viewer_server.thrift

```diff
@@ -250,6 +250,9 @@ service codeCheckerDBAccess {
   // returns the database access handler api version
   string getAPIVersion();
 
+  // returns the CodeChecker version that is running on the server
+  string getPackageVersion();
```

Comment:
Shouldn't we increase the `API_VERSION = '5.0'` to `5.1`?  
We are adding a new API method, this is just an extension and won't break backward compatibility.